instructor/compiler.py

```diff
@@ -353,10 +353,10 @@ def drugs(assignment):
 def concentrations(assignment):
     ret = {}
     for strain_protocol in assignment.strain_treatment.filter(enabled=True):
-        treatment = strain_protocol.treatment
-        concentration = treatment.drug.concentration
-        ret[str(concentration)] = {
-            'name': str(concentration),
+        drug = strain_protocol.treatment.drug
+        concentration = drug.concentration
+        ret[concentration] = {
+            'name': str(concentration) + drug.concentration_unit,
```

Comment:
Instead of using string concatenation here can you use format strings instead? For example `"{concentration}{unit}".format(concentration=concentration, unit=drug.concentration_unit)`. Format strings are nicer for i18n and as a side benefit you don't have to use `str()` each time you want to concatenate a number with a string